gm4_zauber_cauldrons/data/zauber_cauldrons/functions/cauldron/wormhole_targeting/arrival_animation.mcfunction

```diff
@@ -1,11 +1,11 @@
 # @s player consuming wormhole_bottle
-# at @s in depending on call
-# run from cauldron/wormhole_targeting/teleport_to_dimension
+# at @s (after teleport)
+# run from cauldron/wormhole_targeting/choose_dimension
 
 #particles and sound
-particle minecraft:portal ~ ~.6 ~ .25 .25 .25 0 100
-playsound minecraft:entity.enderman.teleport player @a[distance=..8] ~ ~ ~ 1 .3
+particle portal ~ ~.6 ~ .25 .25 .25 0 100
+playsound entity.enderman.teleport player @a[distance=..8] ~ ~ ~ 1 .3
 
 #explosion if no cauldron in target block
 execute align xyz unless entity @e[type=area_effect_cloud,tag=gm4_zauber_cauldron,dx=0,dy=0,dz=0] run summon tnt
-execute align xyz unless entity @e[type=area_effect_cloud,tag=gm4_zauber_cauldron,dx=0,dy=0,dz=0] run playsound minecraft:entity.generic.explode block @s
+execute align xyz unless entity @e[type=area_effect_cloud,tag=gm4_zauber_cauldron,dx=0,dy=0,dz=0] run playsound entity.generic.explode block @s

```

Comment:
Something seems to be off with this arrival animation: 

1. Valid Cauldron at dest. , unloaded:
- **Current Behavior:** Enderman teleport sound, explode sound (no TNT).
- **Expected Behavior:** Enderman arrival particle and teleport sound, no explode sound.

2. No Cauldron at dest. , unloaded: 
- **Current Behavior:** Enderman teleport sound, explode sound, but no TNT is summoned.
- **Expected Behavior:** Enderman teleport sound, explode sound, primed TNT entity.


This might be correlated to the missing arrival particle unless one warps to a cauldron within particle render distance - as in these checks might run at the wrong `at @s` or  simply the wrong time.